project/CommonSettings.scala

```diff
@@ -141,6 +141,7 @@ object CommonSettings {
     "-Xmax-classfile-name",
     "128",
     "-Ywarn-unused",
+    "-Ywarn-macros:after",
```

Comment:
Added somewhere down the line. Will see what breaks when removing this :sweat_smile:

Reply:
Removed